tests/expression_statement/use_strict.js

```diff
@@ -0,0 +1,4 @@
+// Parentheses around expression statement should be preserved in this case.
+("use strict");
+// Ensure no regression.
+"use strict";
```

Comment:
You actually need to have this as the first expression, otherwise it is being parsed as just a string and not a directive. I would recommend making two files